honeycomb/honeycomb.go

```diff
@@ -125,7 +125,7 @@ func honeycombSpan(s *trace.SpanData) Span {
 	}
 
 	if s, e := s.StartTime, s.EndTime; !s.IsZero() && !e.IsZero() {
-		hcSpan.DurationMs = int(e.Sub(s) / time.Millisecond)
+		hcSpan.DurationMs = float64(e.Sub(s) / time.Millisecond)
```

Comment:
I’m by sure this will work right. The inner division might be done as integer division, producing an integer result, which is then cast to float. A test will confirm but you might need to cast each opera down individually rather than just the result.

Reply:
You were right, and now that I'm parsing it properly, I can't successfully find a way to test the result in the e2e test